src/main/webapp/scripts/keywords.js

```diff
@@ -17,4 +17,3 @@ $().ready(function() {
     $("#form-title *").text("Copy Your Tweet Here");
   });
 });
-
```

Comment:
[Nit] it seems like you deleted a newline

Reply:
There were two trailing new lines